src/pages/_document.tsx

```
@@ -33,33 +33,26 @@ export default class MyDocument extends Document<Props> {
           />
           <style
             id={'_goober'}
-            // And defined it in here
             dangerouslySetInnerHTML={{ __html: ' ' + this.props.css }}
           />
         </Head>
         <script
           dangerouslySetInnerHTML={{
```

Biome (2.4.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

(lint/security/noDangerouslySetInnerHtml)